R/html-tweak.R

```diff
@@ -101,13 +101,17 @@ tweak_code <- function(x) {
     xml2::xml_find_all(".//pre[contains(@class, 'r')]") %>%
     purrr::map(tweak_pre_node)
 
-  # Needs to second so have all packages loaded in chunks
-  # <code> with no children (just text)
+  # Identify <code> with no children (just text), and are not ancestors of a
+  # header
+  xpath <- paste0(
+    ".//code[count(*) = 0 and ",
+    "not(ancestor::h1|ancestor::h2|ancestor::h3|ancestor::h4|ancestor::h5)]"
```

Comment:
I assume `not(ancestor::(h1|h2|h3|h4|h5))` doesn't work?

Reply:
Unfortunately it does not.